backend/db/utils.py

```diff
@@ -236,17 +236,22 @@ def chunk_list(input_list: List, chunk_size) -> List[List]:
         yield input_list[i:i + chunk_size]
 
 
-def current_datetime(time_zone=['UTC/GMT', 'EST/EDT'][1]) -> str:
-    """Get current datetime in ISO format as a string."""
+def tz_datetime_str(dt: datetime, time_zone=['UTC/GMT', 'EST/EDT'][1]) -> str:
```

Comment:
## `tz_datetime_str`
Creates timezone-fromatted datetime string. Added to DRY things up a bit.